.github/workflows/pull-request.yml

```
@@ -0,0 +1,39 @@
+      - '**'
+
+
+jobs:
+  CodeRabbit-Approval:
+      name: Validate CodeRabbit.ai Approval
+      runs-on: ubuntu-latest
+      steps:
+        - name: Check CodeRabbit.ai Approval
+          run: |
+            reviews=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+              https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/reviews) 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)